### PR TITLE
Handled promise rejection when failed to subscribe pair

### PR DESCRIPTION
**Pull request by adeacetis**

### CODE REVIEW

File: src/exchange.js [hunk 1]
```diff
@@ -635,6 +639,7 @@ class Exchange extends EventEmitter {
     if (!forceRefreshProducts) {
       try {
         formatedProducts = await readProducts(this.id)
+        console.log(formatedProducts)
```
Comment (Tucsky) [inline src/exchange.js:642]: comment

Comment (adeacetis) [reply src/exchange.js]: I have pushed the changes. Thank you for catching this.

**Comment by adeacetis** [comment]
This one is done as well

**Comment by adeacetis** [comment]
When you are ready @Tucsky

**Comment by Tucsky** [comment]
You should have the right to merge already @adeacetis  
sent you an invite

**Comment by adeacetis** [comment]
Yes, but you didn't approve. That's it :)

**Pull request merged**